docs: Remove old bootstrap node before restoration (#2562)
scality/metalk8s: docs/operation/bootstrap_backup_restore.rst

      --cert-file /etc/kubernetes/pki/etcd/server.crt \
      member remove <etcd_id>

> As MetalK8s do not support multi bootstrap right now, the old bootstrap node

Reviewer: ```suggestion
Since multiple bootstrap nodes are not supported for the moment, the old bootstrap Node
```

> need to be removed before performing the restoration. To do so, run the

Reviewer: ```suggestion
needs to be removed before performing the restoration. To do so, run the
```

> following commands from a working Node with ``master`` role:

Reviewer: ```suggestion
following commands from a working master node:
```

Author: This one was just to match the step just before for etcd

Reviewer: Given our brand new writing guidelines TechPubs provided, we should change both... But OK for this PR, and we'll do a cleanup of all docs before end of Q2 delivery.

.. code::

  # List all nodes to get the node name of the old bootstrap node that need
  # to get removed
  kubectl get node --selector="node-role.kubernetes.io/bootstrap" \
     --kubeconfig=/etc/kubernetes/admin.conf

  # Remove the old bootstrap node (replace <node_name> in the command)

>   kubectl delete node <node_name> --kubeconfig=/etc/kubernetes/admin.conf

Reviewer: In a single command:

```shell
kubectl --kubeconfig=/etc/kubernetes/admin.conf \
    delete nodes --selector="node-role.kubernetes.io/bootstrap" --all
```

Author: Hmm I know (and btw the `--all` is not really needed), but no, it's bit dangerous to me. I really prefer the user to choose it manually

Reviewer: OK
